packages/flutter_tools/test/protocol_discovery_test.dart

```diff
@@ -68,7 +68,7 @@ void main() {
 
       testUsingContext('discovers uri even if logs has ESC Ascii', () async {
         initialize();
-        logReader.addLine('Observatory listening on http://127.0.0.1:3333 \x1b[');
+        logReader.addLine('Observatory listening on http://127.0.0.1:3333\x1b[');
```

Comment:
Presumably the changes above are also made to work on Windows, so let's add some tests of both IPv6 and Windows URLs with the trailing `\x1b[` sequence